gradle.properties

```diff
@@ -18,5 +18,5 @@ ideaEdition = IC
 ideaVersion = 2017.1
 javaVersion = 1.8
 ijPluginRepoChannel = alpha
-version = 17.2.2_2017-SNAPSHOT
+version = 17.2.3
```

Comment:
see https://github.com/GoogleCloudPlatform/google-cloud-intellij/commit/e2696fac201422162ffd32b4b76d33de566271ae

the version should probably be 17.2.3_2017-SNAPSHOT